### PR TITLE
Added Jest icon #1326

### PR DESCRIPTION
**Pull request by nexus0212**
**Issue: #1326**

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Color was picked from the website element class `.headerTitleWithLogo` [1]

---
[1] https://jestjs.io/

### CODE REVIEW

File: icons/jest.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....4-1-2.7-2.3-3L23 0H8.1l3 8.8c-1.3.4-2.2 1.6-2.2 3 0 1 .5 2 1.3 2.5-.3.4-.7.9-1.1 1.2-.8.7-1.6 1.3-2.6 1.6-.8-.6-1.2-1.6-.9-2.4 1.3-.4 2.2-1.6 2.2-3 0-1.7-1.4-3.1-3.1-3.1-1.7 0-3.1 1.4-3.1 3.1 0 .8.3 1.6.9 2.2 0 .1-.1.2-.1.3-.5 1-1 2-1.2 3.3-.4 2.5.3 4.5 1.9 5.6.8.6 1.8.9 2.9.9 1.8 0 3.7-.9 5.5-1.8 1.3-.6 2.6-1.3 3.9-1.6l1.5-.3c1.1-.2 2.2-.3 3.2-.9 1.2-.7 2-1.8 2.2-3.1.2-1 0-2-.4-2.9.2-.4.4-1 .4-1.6zm-1.4 0c0 1-.8 1.8-1.8 1.8s-1.8-.8-1.8-1.8c0-.3.1-.6.2-.9.1-.1.1-.2.2-.3l.2-.2c.1 0 .1-.1.2-.1h1.2c1 0 1.6.7 1.6 1.5zM10 1.4h11l-2.5 7.4c-.1 0-.2.1-.3.1l-2.6-5.4L13 8.9c-.1 0-.2-.1-.3-.1L10 1.4zm5 9.8c-.1-.7-.5-1.2-.9-1.7l1.4-2.9 1.5 3c-.4.4-.7 1-.9 1.6H15zm-3.5-1.1c.1 0 .1 0 0 0H13c.1.1.2.1.2.2.1.1.1.2.2.3.2.3.3.6.3 1 0 1-.8 1.8-1.8 1.8s-1.8-.8-1.8-1.8c.1-.6.7-1.3 1.4-1.5zm-6.8 0c1 0 1.8.8 1.8 1.8s-.8 1.8-1.8 1.8-1.8-.8-1.8-1.8.8-1.8 1.8-1.8zm16.2 6c-.2.9-.7 1.7-1.5 2.1-.8.4-1.7.6-2.7.7-.5.1-1.1.2-1.6.3-1.4.4-2.8 1-4.2 1.7-1.7.9-3.3 1.7-4.9 1.7-.8 0-1.5-.2-2.2-.7-1.5-1.1-1.5-3-1.3-4.1.2-1 .6-2 1.1-2.9 0-.1 0-.1.1-.2.2.1.3.1.5.1-.3 1.5.5 3 1.9 3.7l.3.1.3-.1c1.2-.4 2.3-1.1 3.3-2.1.6-.5 1-1.1 1.4-1.7h.5c1.4 0 2.7-1 3-2.3h1.2c.3 1.3 1.6 2.3 3 2.3.6 0 1.1-.2 1.6-.4.2.7.3 1.3.2 1.8z"/></svg>
```
Comment (ericcornelissen) [inline icons/jest.svg]: Looking at the quality of the icon it seems you've optimized a bit too much (look at the open circles at the bottom)

Comment (ericcornelissen) [inline icons/jest.svg]: It seems there is still a `fill="none ` left, could you remove that?

Comment (ericcornelissen) [inline icons/jest.svg]: Can you try to compound the paths in this SVG? If you need help with this let me know 😄

**Comment by nexus0212** [comment]
I fixed all bugs 💃. @ericcornelissen, I will be grateful to check that everything is okey.

**Comment by ericcornelissen** [comment]
Great work @nexus0212. I have two more questions/suggestions:

1. Can you update the link to either the website (https://jestjs.io/) or repository (https://github.com/facebook/jest) both contain SVG versions of the log as well, and they are more likely to exist in the future than the link you selected.
1. For the color you have selected `#15c213` based on the SVG in the link you provided. However, I feel the brand color of Jest is more red than green, would you agree? For example the red in [this SVG](https://github.com/facebook/jest/blob/master/website/static/img/jest.svg): `#c21325` 🤔

**Comment by nexus0212** [comment]
> 2\. For the color you have selected `#15c213` based on the SVG in the link you provided. However, I feel the brand color of Jest is more red than green, would you agree? For example the red in [this SVG](https://github.com/facebook/jest/blob/master/website/static/img/jest.svg): `#c21325` 🤔

Of course I replace the color. In the first time I sugested the color from typographic logo Jest on the website https://jestjs.io/ 🤨

---
- [x] Updated the link
- [x] Changed color `#15c213` -> `#c21325`

**Comment by ericcornelissen** [comment]
Also, can I give you a tip about committing @nexus0212? Because you updated the commit and force pushed all changes on this PR the related issues now looks like this :arrow_down:. Not necessarily a problem but it gets a bit messy.

image

At least for this project it is easier to just create a new commit. This makes your life easier as you don't have to write comments like this :arrow_down: Instead that information will just be in the commit history. You don't have to care about cluttering the commit history as we use [squash commits](https://help.github.com/en/articles/about-pull-request-merges#squash-and-merge-your-pull-request-commits) for Pull Requests like this 😉 

image

I hope that helps (don't hesitate to ask for clarification). Again, it is just some advice and you're doing great work! It is much appreciated 😃

**Pull request merged**